src/ansys/sherlock/core/project.py

```diff
@@ -1116,3 +1117,311 @@ def update_thermal_maps(self, project, thermal_map_files):
             for error in e.str_itr():
                 LOG.error(error)
             raise e
+
+    def add_thermal_maps(self, project, add_thermal_map_files):
+        """
+        Add thermal map files to a Sherlock project.
+
+        Parameters
+        ----------
+        project : str
+            Name of the Sherlock project to add thermal maps to.
+        add_thermal_map_files : list
+            List of thermal map files consisting of these properties:
+
+                - thermal_map_file : str
+                    Full path to the thermal map file to add.
+                - thermal_map_file_properties : list
+                    List of thermal map properties consisting of these properties:
+
+                        - file_name : str
+                            Name of the thermal file to update.
+                        - file_type : ThermalMapsFileType
+                            Thermal maps file type.
+                        - file_comment : str, optional
+                            Comment to associate with the file.
+                        - thermal_board_side : ThermalMapsBoardSide
+                            Thermal board side.
+                        - file_data : CsvExcelFile|IcepakFile|ImageFile
+                            The properties of the thermal map file to update.
+                        - thermal_profiles : List of str
+                            List of thermal profiles.
+                        - cca_names : List of str, optional
+                            List of CCA names to provide thermal maps for. The default is ``None``,
+                            in which case all CCAs in the project are returned.
+
+        Returns
+        -------
+        int
+            Status code of the response. 0 for success.
+
+        Examples
+        --------
+        >>> from ansys.sherlock.core.launcher import launch_sherlock
+        >>> from ansys.sherlock.core.types.project_types import (
+            BoardBounds,
+            ImageBounds,
+            ImageFile,
+            LegendBounds,
+            LegendOrientation,
+            ThermalBoardSide,
+            ThermalMapsBoardSide,
+            ThermalMapsFileType,
+        )
+        >>> sherlock = launch_sherlock()
+        >>> thermal_map_properties = ImageFile(board_bounds=BoardBounds([
+            (1.0, 2.0),
+            (3.0, 4.0),
+            (1.0, 2.0),
+            (1.0, 2.0)]),
+            coordinate_units="in",
+            image_bounds=ImageBounds(0.0, 0.0, 10.0, 8.0),
+            legend_bounds=LegendBounds(1.0, 2.0, 4.0, 2.0),
+            legend_orientation=LegendOrientation.VERTICAL,
+            max_temperature=50.0,
+            max_temperature_units="C",
+            min_temperature=20.0,
+            min_temperature_units="C"

```

Comment:
Should list the min parameters before the max parameters since that is a normal convention when specifying a range and because that is the order they are specified in the proto file.